python/pyspark/ml/tests/test_algorithms.py

```diff
@@ -86,7 +86,7 @@ def test_raw_and_probability_prediction(self):
         expected_rawPrediction = [-11.6081922998, -8.15827998691, 22.17757045]
         self.assertTrue(result.prediction, expected_prediction)
         self.assertTrue(np.allclose(result.probability, expected_probability, atol=1E-4))
-        self.assertTrue(np.allclose(result.rawPrediction, expected_rawPrediction, atol=1E-4))
+        self.assertTrue(np.allclose(result.rawPrediction, expected_rawPrediction, atol=1))
```

Comment:
Is `1`  the minimum difference?

Reply:
Yup .. 

JDK 8: 

```
[-11.19194106875243,-7.677866573997363,21.280214474039443]
```

JDK 11:

```
[-11.608192299802019,-8.158279986906651,22.177570449962918]
```

Seems multiple floats affects the results while they are roughly correct.

Reply:
I'm not sure where the difference comes from, but it could be subtle differences in randomization or something across the JDKs. If these two tests are the only ones that vary, I think we're OK. I agree with loosening the bound here as these are log-odds, and I suspect the test values were picked just because it's what some previous run spit out (that is, it's too specific)